src/services/CapabilitiesManager.ts

```diff
@@ -169,3 +172,15 @@ function restoreRemoteCapabilities(): RemoteCapabilities {
 
 	return remoteCapabilities
 }
+
+/**
+ * Set user config using provisioning API
+ *
+ * @param configKey - key of the config to set
+ * @param configValue - value to set
+ */
+export async function setUserConfig(configKey: keyof Config[keyof Config], configValue: ConfigValueType): Promise<void> {
```

Comment:
Config has only `call.start-without-media`, whereas configKey is `calls_start_without_media`, so you wouldn't get a match here
Both ConfigValueType and `keyof Config[keyof Config]` (in my IDE) already evaluates to never and don't make much sense

I'd suggest to not overcomplicate it here inside of Manager, but keep both functions in settingsService:
- generic setUserConfig() - I'd also pass 'spreed' separately, in case we'll want to change something in another app in future from Talk / Talk Desktop
- more specific setStartWithoutMedia() - here you know that value is boolean, and the rest is already fixed